compiler-plugin/src/main/kotlin/arrow/meta/plugins/proofs/phases/resolve/ProofResolution.kt

```diff
@@ -148,7 +148,9 @@ fun FunctionDescriptor.asProof(): Proof? =
   extensionReceiverParameter?.type?.let { from ->
     returnType?.let { to ->
       val annotationArgs = annotations.first().allValueArguments
-      val value: ConstantValue<*>? = annotationArgs[Name.identifier("of")]
+      val value: ConstantValue<*> = annotationArgs.getValue(Name.identifier("of"))
+      val coerce: ConstantValue<Boolean> = annotationArgs.getValue(Name.identifier("coerce")) as ConstantValue<Boolean>
```

Comment:
Same here, we can't know coerce is gonna be there, can we?

Reply:
Yeah, actually I was seeing errors in the logs because of that..